PROTO_tests/tests/series_test.py

```diff
@@ -0,0 +1,150 @@
+import pandas as pd
+import pytest
+from pandas.testing import assert_frame_equal
+
+import arkouda as ak
+
+
+class TestSeries:
+    def test_series_creation(self):
+        idx = ak.arange(3)
+        for val in ak.arange(3), ak.array(["A", "B", "C"]):
+            ans = ak.Series(data=val, index=idx).to_list()
+            for series in (
+                ak.Series(data=val, index=idx),
+                ak.Series(data=val),
+                ak.Series(val, idx),
+                ak.Series(val),
+                ak.Series((idx, val)),
+            ):
+                assert isinstance(series, ak.Series)
+                assert isinstance(series.index, ak.Index)
+                assert series.to_list() == ans
+
+        with pytest.raises(TypeError):
+            ak.Series(index=idx)
+
+        with pytest.raises(TypeError):
+            ak.Series((ak.arange(3),))
+
+        with pytest.raises(TypeError):
+            ak.Series()
+
+        with pytest.raises(ValueError):
+            ak.Series(data=ak.arange(3), index=ak.arange(6))
+
+    def test_lookup(self):
+        for val in ak.arange(3), ak.array(["A", "B", "C"]):
+            s = ak.Series(data=val, index=ak.arange(3))
+
+            for key in 1, [0, 2], ak.Index([1]), ak.Index([0, 2]):
+                lk = s.locate(key)
+                assert isinstance(lk, ak.Series)
+                key = ak.array(key) if not isinstance(key, int) else key
+                assert (lk.index == s.index[key]).all()
+                assert (lk.values == s.values[key]).all()
+
+            # testing multi-index lookup
+            mi = ak.MultiIndex([ak.arange(3), ak.array([2, 1, 0])])
+            s = ak.Series(data=val, index=mi)
+            lk = s.locate(mi[0])
+            assert isinstance(lk, ak.Series)
+            assert lk.index.index == mi[0].index
+            assert lk.values[0] == val[0]
+
+            # ensure error with scalar and multi-index
+            with pytest.raises(TypeError):
+                s.locate(0)
+
+            with pytest.raises(TypeError):
+                s.locate([0, 2])
+
+    def test_add(self):
```

Comment:
This also should be tested for all `DTYPES`.

Reply:
Updated